src/assets/thirdPartyServices/snet/time_series_anomaly_discovery/index.js

```diff
@@ -17,6 +17,8 @@ import Slider from "@material-ui/lab/Slider";
 import { Chart } from "react-google-charts";
 import Tooltip from "@material-ui/core/Tooltip";
 
+import { TSAnomalyDetection } from "./timeSeriesAnomalyDetection_pb_service";
```

Comment:
Can we verify that this stub is properly generated. Something dosen't look right.